src/main/resources/db/migration/V1_46__ADD_CMS_CONTACT_DEFINITIONS.sql

```diff
@@ -0,0 +1,14 @@
+
+INSERT INTO app.adjustment_reason (contact_code, contact_description, category_id, points)
+VALUES ('CMS40', 'CMS - Assistance with Assessments', 1, 0),
```

Comment:
Is zero a valid number of points? Is it the correct default?

Reply:
I believe so, can always create a NART report with a CMS entry with this code in to see